src/deps/binary/base64-browser.js

```diff
@@ -3,6 +3,9 @@ var thisAtob = function (str) {
 };
 
 var thisBtoa = function (str) {
+  if (/[\u0080-\uFFFF]/.test(str)) {
+    str = unescape(encodeURIComponent(str));
+  }
```

Comment:
is the `if` statement necessary? I think we can just call `unescape(encodeURIComponent())` every time, right?

Actually now that I think about it, is it more reasonable to ask users to do this themselves?